OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -37,9 +37,2085 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask29SalArsima"/>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i1.2.29.6">
+                                        <locus target="#45rc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask29SalJohn"/>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i1.2.29.7">
+                                        <locus target="#45rc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask29SalJohn2"/>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i1.2.29.8">
+                                        <locus from="45rc" to="45va"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask29SalChrist"/>
+                                    </msItem>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.2.30">
+                                    <locus from="45va" to="46vc"/>
+                                    <title type="complete" ref="LIT4035SenkessarDL#Mask30"/>
+                                    
+                                    <msItem xml:id="ms_i1.2.30.1">
+                                        <locus from="45va" to="46ra"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask30Athanasius"/>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i1.2.30.2">
+                                        <locus target="#46ra"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask30Gorgoryos"/>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i1.2.30.3">
+                                        <locus target="#46ra"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask30John"/>
+                                    </msItem>
+                                    
+                                    <msItem xml:id="ms_i1.2.30.4">
+                                        <locus from="46rb" to="46rc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Mask30Salusi"/>
+                                        <explicit xml:lang="gez">
+                                            <locus target="#46rc"/>
+                                            ወሶበ፡ ሰምዓ፡ ሊአ፡ ጳጳሳት፡ ፈቀደ፡ ይንሥአ፡ ለይእቲ፡ ቆብዕ፡ ይትባረክ፡ እምኔሃ<hi rend="rubric" rendition="#partialRubric">፨</hi>
```

Comment:
This is the explicit of the commemoration of the last day of Maskaram. I wonder whether it makes sense to provide the explicit of the last commemoration notice. it is not the explicit of the month (unlike the beginning of the month, which refers to the whole month and not only to the first commemoration notice.  
As you provide the explicit of the month under "div @ edtion", I twould suggest that toghether with the incipit of the month (quoted under ms item) the text is sufficiently described and made identifiable.

Reply:
Just to ensure that I understood you well: You suggest only giving the beginning of the month and the subscription, not the incipits and explicits of the first and last commemoration?
The other months don't have such an elaborated beginning, so I would go on giving an incipit of the first commemoration there. Also regarding the subscription - I don't know the Second Recensions well enough to attest to their standardisation. But in general, there is more flexibility in these than in the commemorations, so I think it would still be better to give at least one commemoration incipit and explicit per month, to have a basis for a first philological assessment.

Reply:
I thought that every month has a kind of prologue, but if not, hm.
However, isn't it odd to provide the beginning of one text and the end of another one? We consider each commemoration notice as an independend text unit. That was my reasoning behind it. But it was just a suggestion. You are the Senkessar expert. I am fine with whatever you think is right. :)

Reply:
If you provide the incipits and explicits of the first and last commemoration notice this would make more sense to me (but as I said, it is entitely up to you as trusted expert)

Reply:
I agree that it is odd! Incipit and explicit of the first and last commemoration seem reasonable to me.